handler/commentHandler_test.go

```diff
@@ -803,5 +805,57 @@ func Test_getCommandValue(t *testing.T) {
 			}
 		})
 	}
+}
 
+func Test_createIssueCommen(t *testing.T) {
```

Comment:
Test_createIssueCommen => Test_createIssueComment

Reply:
Good attention to detail 💯 Fixed